modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -608,6 +608,26 @@ const questionnaireStyle = theme => ({
       padding: theme.spacing(1.5, 0.5),
       borderRadius: theme.spacing(0.5),
     },
+    descriptionPopover: {
+      "& .MuiPopover-paper" : {
+        padding: theme.spacing(3),
+        width: theme.spacing(87),
+        height: theme.spacing(31),
+      }
+    },
+    descriptionPopoverButton: {
+      float: "right",
+      marginTop: theme.spacing(3),
+      marginLeft: theme.spacing(2),
+    },
+    selectionChild: {
+      display: "block"
+    },
+    selectionDescription: {
+      fontSize: "0.75rem",
```

Comment:
Once this is wrapped in a `Typography` component (see other comment), `fontSize` should be switched to  
```suggestion
      fontSize: "inherit",
```